policies/project-director-election-process.md

```diff
@@ -0,0 +1,60 @@
+# Project Director Election Process
+
+## Aims
+
+The purpose of this policy is to outline a clear and efficient process for electing new Project Directors to represent the Rust project on the board of the Rust Foundation. The process aims to achieve the following goals:
+
+* Select candidates who fulfill the outlined candidate criteria, focusing on shared power and new leadership rather than solely prioritizing the best candidate.
+    * Strive for tolerance and inclusivity, aiming for the overlap of tolerance rather than the overlap of preference.
+* Conduct the selection process in a time-efficient manner.
+* Incorporate input from a wide range of stakeholders across the entire Rust project.
+* Ensure a diverse pool of qualified candidates for selection.
+* Create a psychologically safe environment that values minority perspectives and encourages positive feedback, encouragement, and tolerance rather than preference or criticism.
+* Ensure participants feel positive about their interactions and the execution of the process.
+
+## Description
+
+### Setup
+
+* The council selects a facilitator by consent:
+    * Any team member can suggest a facilitator or volunteer themselves.
+    * The facilitator must not be a nominee in the election and must be someone everyone trusts to set aside personal preferences during the election process.
+    * **Aims of the facilitator role**
+        * Clearly documenting and communicating the process and announcing and supporting its execution.
+        * Tracking, reviewing, and documenting the effectiveness of the process.
+* Review or Define Role Description ([example](https://github.com/rust-lang/rfcs/pull/3392#issuecomment-1505697944))
+    * Aims
+    * Time Investment
+    * Responsibilities
+    * Qualifications
+    * Term
+
+### Pre-Meeting Coordination
+
+* Gather nominations
+    * Council representatives collaborate with their subteams to collect nominations on behalf of their branch of the organization tree.
+    * Subteams can utilize any process they prefer, but we recommend following a similar approach to the process outlined in the "Election Meeting" section below
+* Share nominations publicly
+    * Each team representative must publicly share their team's nominations at least ten days before the final election meeting. This allows other teams to share feedback or objections about candidates their team didn't initially consider with their representative.
+    * This 10-day period is equivalent to the Change Round in the "Election Meeting" step; representatives may change their nominations in advance of the meeting
+
+### Election Meeting
+
+* Share nominations
+    * Write down nominations first[^1].
+    * Use a round format (e.g. [round format](https://www.sociocracyforall.org/on-rounds/))
+    * Nominations must include supporting reasoning 
+* Change Round
+    * During the change round, every team member will say who they nominate now and share reasons for their change if that applies.
```

Comment:
(Assuming we go with batch selection) do we want to emphasize that "who" means the set of new Project Directors rather than an individual?

Reply:
+1 it is ambiguous whether there is a 1 to 1 or a 1 to many correspondence between meeting participants and nominees. Can a meeting participant have many nominees? Should they be striving to nominate a list that would fill all open slots? Can they nominate more than the number of open slots?